source/nodejs/adaptivecards-site/pages/_posts/2020/Updated-Versioning-Policy.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Updated Versioning and Release policy
+subtitle: Packaging and release software can be complicated, here's what we're doing
+date: 2020-07-05
+github_username: matthidinger
+twitter: matthidinger
+---
+
+## THIS IS A DRAFT
+
+This is an early proposal and we may change these decisions before publishing this post :)
+
+# Updated Versioning and Release policy
+
+Versioning, releasing, and marketing software can be complicated. We're making some changes this year to clarify a few things that should help our customers and developers better understand **what the versions mean**, and **when to expect updates**.
+
+## TL;DR
+
+- Adaptive Cards is an umbrella term for multiple interconnected products: the Adaptive Card schema, the renderer SDKs, the templating schema and templating SDKs, and the card designer.
+- Each of those products exist on different platforms and have their own release cycles.
+- We will follow Semantic Versioning (SemVer) where possible, which means if any individual product has a major update or API breaking change, we will increment the version as appropriate for that product. As such, our different products will necessarily have different versions (e.g., the latest JavaScript rendering SDK may be version `2.0`, but the latest Adaptive Card Schema is `1.3`).  
```

Comment:
Are there places where we plan or envision to not follow semantic versioning at all? What about "We follow Semantic Versioning as closely as possible. This means ...."